src/test/java/de/tum/in/www1/artemis/ExamIntegrationTest.java

```diff
@@ -931,6 +931,8 @@ public void testGetExamWithOptions() throws Exception {
         assertThat(exam3.getExerciseGroups()).hasSize(exam.getExerciseGroups().size());
         assertThat(exam3.getExerciseGroups().get(0).getExercises()).hasSize(exam.getExerciseGroups().get(0).getExercises().size());
         assertThat(exam3.getExerciseGroups().get(1).getExercises()).hasSize(exam.getExerciseGroups().get(1).getExercises().size());
+        assertThat(exam3.getNumberOfRegisteredUsers()).isNotNull();
+        assertThat(exam3.getNumberOfRegisteredUsers()).isEqualTo(1);
```

Comment:
```suggestion
        assertThat(exam3.getNumberOfRegisteredUsers()).isNotNull().isEqualTo(1);
```
This could a one-liner. You could also remove the `isNotNull()` completely because it's already covered by the `isEqualTo()`-check.

Reply:
Thanks!

Reply:
This is the same at the three other places in the tests.